cmd/aro/rp.go

```diff
@@ -52,6 +52,7 @@ func rp(ctx context.Context, log, audit *logrus.Entry) error {
 	if _env.IsLocalDevelopmentMode() {
 		keys = []string{
 			"PULL_SECRET",
+			env.OIDCStorageAccountName,
```

Comment:
Nit: Why not initialize `keys ` list with `env.OIDCStorageAccountName` outside `if ` condition since it's used in both `if ` & `else ` part

Reply:
I agree with this refactor, but for the sake of fixing the broken deployment I suggest we tackle this in a follow-up PR.